dolphinscheduler-ui/src/views/ui-setting/index.tsx

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useI18n } from 'vue-i18n'
+import { NForm, NFormItem, NSelect } from "naive-ui";
+import { defineComponent } from "vue";
+import { useLogTimerStore } from '@/store/logTimer/logTimer'
+import type { logTimer } from '@/store/logTimer/types'
+import cookies from 'js-cookie'
+ 
+// Update LogTimer store when select value is updated
+const handleUpdateValue = (logTimer: logTimer) => {
+    const logTimerStore = useLogTimerStore()
+    logTimerStore.setLogTimer(logTimer)
+    cookies.set('logTimer', String(logTimer), { path: '/' })
```

Comment:
Why set the logTimer into the cookies? Does the backend side need it?

Reply:
Hi. We might not need this. I will remove it.

Reply:
We might not need this. Removed in https://github.com/apache/dolphinscheduler/pull/11226/commits/145bfd49b3bba292f216eaf0602bc828d0fe8024.